backend/sozisel/lib/sozisel/model/events.ex

```diff
@@ -50,7 +69,12 @@ defmodule Sozisel.Model.Events do
     Event.create_changeset(event, attrs)
   end
 
+  # FIXME: I don't remember why we added that to be honest...
```

Comment:
This function was helpful in preparing data for the participant depending on the type of event. In the case of Quiz participant did not receive a set of correct answers. But I heard that in #100 @jmysliv removed this logic after consulting with you, so I don't know if such functions will be needed in the future.